src/openMVG/multiview/rotation_averaging_l1.cpp

```diff
@@ -329,7 +329,7 @@ struct Link {
 };
 typedef std::queue<Link> LinkQue;
 
-#ifdef HAVE_BOOST
+#ifdef OPENMVG_ROTATION_AVERAGING_WITH_BOOST
```

Comment:
can't find where it is defined...
Either we need another option in the main cmakelists depending on boost or boh...?